miniconf.py

```diff
@@ -32,27 +32,53 @@ def __init__(self, client, prefix):
             client: A connected MQTT5 client.
             prefix: The MQTT toptic prefix of the device to control.
         """
+        self.uuid = uuid.uuid1()
+        self.request_id = 0
         self.client = client
         self.prefix = prefix
         self.inflight = {}
         self.client.on_message = self._handle_response
         self.client.subscribe(f'{prefix}/response/#')
+        self.logger = logging.getLogger(__name__)
```

Comment:
Could you elaborate why you don't like the module-level logger? It is [standard practice](https://github.com/python/cpython/blob/main/Lib/asyncio/log.py).
This doesn't help with distinguishing `Miniconf` instances.

Reply:
I moved the logger into `Miniconf` because variables outside of objects are globals. Since we really didn't need the logger to be global (as it's only used within `Miniconf`), I moved it into `Miniconf` to satisfy PyLint. Other than that, there's no particular reason and I don't believe it has a practical implication on end-users.

Reply:
But there are so many global variables. Everything from and inside imported modules to classes and their attributes and functions. A logger to me is **the** paradigmatic global resource in the same sense that `print()` accesses a global resource.
I would buy it if there is a custom formatter that adds e.g. `id(self)` to the logging messages. But that doesn't seem needed.

I have to say I'm disappointed by PyLint here ;)

Reply:
Maybe `flake8` is a bit better.
A slightly more technical reason for a single global logger is resource usage: the case where I have 100 Stabilizer in the lab and I control them with one `Miniconf(prefix, broker)` each. I don't need 100 `Logger` for that.